src/components/ReadmePage.tsx

```diff
@@ -25,35 +26,22 @@ class ReadmePage extends React.Component<
     html: "<p>Loading...</p>",
   }
 
-  async getReadmeHTML() {
-    try {
-      const readmeTextRes = await fetch(this.props.data.readmeURL);
-      if (!readmeTextRes.ok) throw Spicetify.showNotification(`Error loading README (HTTP ${readmeTextRes.status})`);
-
-      const readmeText = await readmeTextRes.text();
-
-      const postBody = {
-        text: readmeText,
-        context: `${this.props.data.user}/${this.props.data.repo}`,
-        mode: "gfm",
-      };
-
-      const readmeHtmlRes = await fetch("https://api.github.com/markdown", {
-        method: "POST",
-        body: JSON.stringify(postBody),
-      });
-      if (!readmeHtmlRes.ok) throw Spicetify.showNotification(`Error parsing README (HTTP ${readmeHtmlRes.status})`);
-
-      const readmeHtml = await readmeHtmlRes.text();
-
-      if (readmeHtml == null) {
+  getReadmeHTML = async () => {
```

Comment:
Did you test the functions that I had the "TODO: test this" on?

Reply:
I tested everything before removing those comments yeah